apisix/balancer/ewma.lua

```diff
@@ -0,0 +1,240 @@
+--
```

Comment:
@redynasc Did you write this code yourself, or copied it from another open source project? If it is copied, the source and license information MUST be clearly stated.

Reply:
This code file is copied from https://github.com/kubernetes/ingress-nginx/blob/master/rootfs/etc/nginx/lua/balancer/ewma.lua, so we MUST keep the oirgin license and copyright, this is respect for other open source projects and a rule that Apache project must follow.